question 7.py

```diff
@@ -0,0 +1,31 @@
+f=open('poem.txt','r').read()
```

Comment:
Upload poem.txt too

Reply:
oh cool

Reply:
I need to type it lmao, I used another sentence to check

Reply:
Copy it from the docx

Reply:
its a picture
right?

Reply:
https://products.groupdocs.app/viewer/view?file=881b5041-23db-480a-9dc3-c2114891b136/Grade%20XII%20Practical%20list%20-term%201.docx

Reply:
Check my commit, I copied it from the docx

Reply:
oh cool

Reply:
cool done